missions/uuv-ex1.xml

```diff
@@ -37,11 +36,12 @@
   <origin_length>10</origin_length>
 
   <entity_interaction>SimpleCollision</entity_interaction>
+  <metrics tags="REAL">SimpleCollisionMetrics</metrics>
 
   <enable_screenshots min_period="1.0" start="8.3" end="15.3">false</enable_screenshots>
 
-  <network>GlobalNetwork</network>
-  <network>LocalNetwork</network>
+  <network tags="REAL">GlobalNetwork</network>
```

Comment:
While we are updating the tags, can we unify this convention with what is done for entity? In other words, can we update it from 

```[xml]
  <entity>
    <name>uuv_entity</name>
  </entity>
``` 

to 

```[xml]
<entity tags="uuv_entity">
</entity>
 ```

Reply:
Yes, I'll use entity tags instead.